src/tests/v1/test_drifting_metrics.py

```diff
@@ -17,7 +20,10 @@ def test_drifting_metric_get_model_multi_class_all(client):
 
 @pytest.mark.order(get_order_number("drifting_metrics_get_model_binary_all"))
 def test_drifting_metric_get_model_binary_all(client):
-    response = client.get(f"/v1/models/{state.model_binary['id']}/drifting_metrics")
+    response = client.get(
+        f"/v1/models/{state.model_binary['id']}/drifting_metrics",
```

Comment:
I just noticed this. we shouldn't have underscores in our endpoints. It should be `drifting_metrics`